pom.xml

```diff
@@ -60,7 +60,45 @@
         <dockerHubOrganization>cafapi</dockerHubOrganization>
         <dockerCafImagePrefix>${dockerImagePrefix}${dockerHubOrganization}${dockerOrgSeperator}</dockerCafImagePrefix>
         <dockerProjectVersion>${dockerVersionSeperator}${project.version}</dockerProjectVersion>
+        <logbackVersion>1.2.3</logbackVersion>
     </properties>
+    <dependencies>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-access</artifactId>
+            <version>${logbackVersion}</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
```

Comment:
How come this has been suddenly added?

Reply:
The logback-classic is the transitive dependency from com.github.tomcat-slf4j-logback:tomcat9-slf4j-logback:jar:9.0.26 with version 1.3.0-alpha4. But this should be 1.2.3. So overriden with 1.2.3.